portal-api/src/v1/routes.js

```diff
@@ -289,11 +289,10 @@ authRouter
   );
 
 authRouter
-  .route('/banks/:bankId/utilisation-report-download/:_id')
+  .route('/banks/:bankId/utilisation-report-download/:id')
   .get(
     validateUserHasAtLeastOneAllowedRole({ allowedRoles: [PAYMENT_REPORT_OFFICER] }),
     bankIdValidation,
-    mongoIdValidation,
```

Comment:
Could we replace this with an `sqlIdValidation` middleware?

Reply:
Yeah I'm kind of debating the point of this in the first place, and checking if the id passed in is a number. I would hope our ORM and typing would prevent anything malicious making it's way to the db call. Either way we certainly don't need to check it is an integer and in the controller and the middleware, so perhaps I will just move the check here if we do think it is important?